recipes/dpgen/add_version.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.py b/setup.py
```

Comment:
What is the purpose of the version patch here?

Reply:
The origin program uses `setuptools-git-version` to read the version from the git tag, but there is no git tags in a `.tar.gz` file. I have to add the version into `setup.py`.